productpage.html

```diff
@@ -277,5 +379,29 @@
         <div class="footer-copywrite">Copyright&copy; 2020 PeaTea</div>
       </div>
     </div>
+    <script>
+      const description = document.querySelector(".description-tab");
```

Comment:
You can also give all tabs one common class, like `.tab`. That way you can remove active class from all tabs at once.